payments-app-extension-credit-card/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,28 @@
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
+api_version = "2021-07"
+
+[[extensions]]
+name = "{{ name }}"
+type = "payments_app_extension"
```

Comment:
I think the specification linked in the PR description is of type `payments_app_credit_card`

Reply:
the believe the type should correspond to the specification identifier: https://github.com/Shopify/shopify/pull/469870/files#diff-348901488c164aca2dace245d34a0ad18c26750b66693a10b71c5012974dc17fR6

Reply:
Oh, I see! From the PR description I thought the management class is `PaymentsPartners::PaymentsAppExtensions::CreditCardPaymentsAppExtension`, https://github.com/Shopify/shopify/blob/main/components/payment_processing/payments_partners/app/models/payments_partners/payments_app_extensions/credit_card_payments_app_extension.rb, which I found it connected only with  ["Payments App - Credit Card"](https://github.com/Shopify/shopify/blob/470f17bf4a59cfc8bdd4e49126b19c2408a5ba59/components/payment_processing/payments_partners/config/initializers/app_extensions.rb#L37) specification.